src/render-style-transfer.py

```diff
@@ -45,16 +45,15 @@ def randomly_choose(list_of_stuff):
     return list_of_stuff[perm]
 
 
-def train(f_style, f_psi, trainloader, trainset, keep_logs=False):
-    file_name = time.time_ns()
+def train(f_style, f_psi, trainloader, trainset, loss_file_name, keep_logs=False):
```

Comment:
shouldn't have to pass in both dataloader and dataset?  I don't see where trainset is being used here.

Reply:
yeah i think you're right